tools/bios-networking

```diff
@@ -29,11 +29,22 @@
 
 # to ensure REST API call at least replies before actually changing the network
 sleep 3
-# On production boxes we have 2 or 3 renamed LAN interfaces
-for L in 1 2 3 ; do ( ifdown --force LAN$L ; sleep 5; ifup --force LAN$L ) & done
-# Support not-remapped devices, including those in a container environment
-# and/or hot-plugged devices in USB ports
-for L in 0 1 2 3 ; do ( ifdown --force eth$L ; sleep 5; ifup --force eth$L ) & done
+
+# restart all network interfaces except lo and kill dhcp client too
+for IFACE in `ls -1 /sys/class/net/`; do
+    [ "${IFACE}" -eq "lo" ] && continue
+
+    (
+    ifdown --force $IFACE ;\
+    if [ -f /run/udhcpc.${IFACE}.pid ]; then \
+        kill -9 `cat /run/udhcpc.${IFACE}.pid`; \
```

Comment:
As noted subsequently, this `kill -9` is likely responsible for not-unconfiguring the NTP client from using a DHCP-provided NTP server instead of the stashed system default. So while DHCP client is no longer running, the NTP client still polls DHCP's NTP server rather than possibly a better one e.g. configured manually (via web-gui) or a pool.ntp.org replica (OS default).